common/time.hpp

```diff
@@ -15,4 +15,12 @@ static inline Time timestamp() {
     return (Time)time.tv_sec * 1000000 + (Time)time.tv_usec;
 }
 
+static inline RJ::Time SecsToTimestamp(double secs) {
```

Comment:
Could you just add a quick comment to doc what these functions do? Other than that. I think its good.